xwiki-platform-core/xwiki-platform-livedata/xwiki-platform-livedata-webjar/src/main/vue/layouts/table/LayoutTable.vue

```diff
@@ -75,7 +75,8 @@
             v-for="(entry, idx) in entries"
             :key="`table-${logic.getEntryId(entry)}-${idx}`"
             :entry="entry"
-            :data-index="idx"
+            :data-livedata-entry-index="idx"
```

Comment:
I know we really have clear practice on this, but wdyt of adding a comment with the "since" version for the two values, as they will become part of the API?

Reply:
Any advice on the format to use? AFAICS I cannot add a comment in the middle of the properties here?